src/store/reducers/rulesReducer.js

```diff
@@ -154,11 +154,12 @@ export const getRulesPendingStatus = state =>
   getPending(getRulesReducer(state), epics.actionTypes.fetchRules);
 export const getRules = createSelector(
   getEntities, getItems, getActiveDeviceGroupId, getActiveDeviceGroupConditions,
-  (entities, items, deviceGroupId, deviceGroupConditions = []) =>
+  (entities, items, deviceGroupId, deviceGroupConditions = [], includeDeleted = false) =>
     items.reduce((acc, id) => {
       const rule = entities[id];
       const activeDeviceGroup = deviceGroupConditions.length > 0 ? deviceGroupId : undefined;
-      return (rule.groupId === activeDeviceGroup || !activeDeviceGroup)
+      return ((rule.groupId === activeDeviceGroup || !activeDeviceGroup) &&
+             (!rule.deleted || includeDeleted))
```

Comment:
I think this conditions should either be one line with the return or one line after the return.

```
return ( ... conditions ... )
  ? true
   : false

OR

return
  ( ... conditions ... )
    ? true
    : false
```